fvtest/compilertriltest/TernaryTest.cpp

```diff
@@ -0,0 +1,222 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2019 IBM Corp. and others
```

Comment:
Is the first copyright year 2018 because some of the code was copied from other files with this copyright date?

Reply:
Fixed.